weltyt.py

```diff
@@ -122,7 +122,10 @@ def buy_lands(lands):
     },
 }
 
+Help = {"show_my_resources": "Shows you what resources you have exactly in this time.", "buy": "shows what is possible to buy in this round.", "Plants": "Opens posibility of buying plants.", "Lands": "Opens posibility of buying lands."}
```

Comment:
Why the capital letter? It's not a class definition.

Reply:
But help is function in python, and capital letter is to programme could distinguish between dictionary and function.

Reply:
Nope, `Help` is not a function, it's a dictionary.
Even if it was a function, it still would be all small characters.
See PEP-8 naming conventions: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables